subsystems/SolenoidSubsystem.java

```diff
@@ -112,8 +64,8 @@ public SolenoidSubsystem(boolean isInverted, DoubleSolenoid... solenoids) {
 	 * @param solenoids
 	 *                  Double solenoids of the system
 	 */
-	public SolenoidSubsystem(DoubleSolenoid... solenoids) {
-		this("SolenoidSubsystem", solenoids);
+	public SolenoidSubsystem( DoubleSolenoid... solenoids) {
```

Comment:
format: remove extra space

Reply:
I think this issue is present in another PR too... we should be careful not to merge the same changes multiple times.

Reply:
see #245.